capstone/src/test/java/com/google/sps/objects/PostTest.java

```diff
@@ -177,12 +177,7 @@ public void removeLikeMultipleTest() {
   @Test
   public void toAndFromPostEntityTest() throws Exception {
     Entity entity = post.toEntity();
-    datastore.put(entity);
-
-    Key postKey = KeyFactory.createKey("Post", POST_ID);
-    Entity retreivedPost = datastore.get(postKey);
-
-    Post returnedPost = Post.fromEntity(retreivedPost);
+    Post returnedPost = Post.fromEntity(entity);
```

Comment:
Not if this comment is relevant anymore, but I just pulled from master and ran the old version of this test, and it was failing. Changing line 64 of post.toEntity to `Entity entity = new Entity("Post", this.postId);` would fix the test but caused issues with being unable to add posts to a group, so probably not the right solution. I guess as long as the test passes like this then you're good!